project/LinkerdBuild.scala

```diff
@@ -142,7 +142,10 @@ object LinkerdBuild extends Base {
       .dependsOn(core, Router.core)
       .withTests()
 
-    val all = aggregateDir("telemetry", core, commonMetrics, tracelog)
+    val requestLog = projectDir("telemetry/request-log")
+      .dependsOn(admin, core, Router.core)
```

Comment:
"tracelog" actually logs traces, whereas this module does not. Also, it's not named consistently with e.g. tracelog.  Any ideas for how to name this to more clearly differentiate it?